### DISCUSSION TITLE
Fetch Full Article - Swipe Down

### DISCUSSION THREAD

**Discussion by Mench84** [spacecowboy/Feeder#265] [Ideas]
I think a great feature that Inoreader has is the ability to swipe down when at the top of an article to fetch the full article contents. I don't mind the icon that Feeder currently uses but sometimes it's pain to stretch my short stubby thumb in order to reach the toggle. Swiping down just feels more fluid. Would love to see that feature in Feeder. Thanks.

**Comment by spacecowboy** [comment]
You know you can configure full text by default for the feed right? then not even a swipe is needed

**Reply by Mench84** [reply]
Yeah, sure do, which is what I've been doing for new feeds that I've added individually. Are you able to toggle that for an OPML import? I imported a large list of feeds that way and have not taken the time to go back and toggle that feature for those feeds. Either way, love the app. Keep up the great work!

**Reply by spacecowboy** [reply]
no but you can manually edit the opml.

export opml from the app and you can see how the setting can be set in opml
